scribble-lib/scribble/lp/lang/common.rkt

```diff
@@ -41,25 +40,24 @@
        (let loop ([block (get-chunk main-id)])
          (append-map
           (lambda (expr)
-            (if (identifier? expr)
-                (let ([subs (get-chunk expr)])
-                  (if (pair? subs)
-                      (begin (set! chunk-mentions (cons expr chunk-mentions))
-                             (loop subs))
-                      (list (shift expr))))
-                (let ([subs (syntax->list expr)])
-                  (if subs
-                      (list (restore expr (loop subs)))
-                      (list (shift expr))))))
+            (cond
+              [(identifier? expr)
+               (define subs (get-chunk expr))
+               (cond
+                 [(pair? subs)
+                  (set! chunk-mentions (cons expr chunk-mentions))
+                  (loop subs)]
+                 [else (list (shift expr))])]
+              [else
+               (define subs (syntax->list expr))
+               (if subs (list (restore expr (loop subs))) (list (shift expr)))]))
```

Comment:
Kind of wish the `if` were on its own line here. Not sure if this is already accounted for in a pending change elsewhere, tho!

Reply:
Sorawee and I have some thoughts on how to address this with some future `fmt` changes. But for now I'd like to just let it be, since I don't think it's worth blocking the rest of the improvements on waiting for that.

Reply:
okay.